modules/changes-logs/classes/class-changes-handle-wp-log-in-out.php

```
@@ -0,0 +1,235 @@
+            'currentuserroles' => $old_user_roles,
+        );
+        Changes_Logs_Logger::log_change( 1595, $log_data );
+    }
+
+    /**
+     * User has requested a password reset.
+     *
+     * @param object $errors Current WP_errors object.
+     * @param object $user   User making the request.
+     */
+    public static function callback_change_user_requested_pw_reset( $errors, $user = null ) { //phpcs:ignore --NOSONAR - requires params.
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$errors'. (undefined)

(UnusedFormalParameter)